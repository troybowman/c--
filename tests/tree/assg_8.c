/* type consistency errors in assignments */

char x[4];
int y[5];

extern void print_int(int x);

void func1(void)
{
  x = 6;
}

void main(int args[])
{
  int x;
  x = 6;
  x = main;
  x = print_int;
  y[3] = y[3];
  main = x;
  args = args;
  print_int = 4;
  y[3] = x;
  x = args;
  x = y;
  y[6] = y[6];
  y[6] = y;
  y[main] = 4;
  y[y] = 4;
}
