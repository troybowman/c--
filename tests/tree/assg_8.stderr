error: invalid operand types for assignment, line 10
syntax error, line: 17
error: invalid operand types for assignment, line 18
syntax error, line: 20
error: invalid operand types for assignment, line 21
error: invalid operand types for assignment, line 22
error: invalid operand types for assignment, line 24
error: invalid operand types for assignment, line 25
error: invalid operand types for assignment, line 27
syntax error, line: 28
error: expression for array index is not of integer type, line 29
